### ISSUE TITLE
Issue with 4.8.2 - Oracle use tries to download beta build (unusable for M1 Mac)

### ISSUE THREAD

**Issue by ZebulonPi** [TablePlus/TablePlus#2786]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): Oracle (version unknown)

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): build 436

3. The steps to reproduce this issue: trying to connect to a pre-existing Oracle connection gives the message "Do you want to download the build which support Oracle". Hitting 'Download' tries to load BETA Build 439, even though I have 'beta builds' unchecked. As I have an M1 Mac, I can't use this version... and the cycle starts again. Oracle WAS working before updating to 4.8.2.

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
Hi @ZebulonPi I don't understand your issue. If you use Oracle, the only build you can use is build 439.
- It's intel only because Oracle does not give us the ARM-M1 driver.
- You must check the "receive beta builds", as it is only available in the Beta build.
If you are using M1 MacBook then you must install rosetta 2.

Please let me know if you are still having any issues.

**Comment by ZebulonPi** [comment]
I think I got confused by the beta build wording, with the "If you want M1 support use the main build" verbiage. I didn't know the Oracle driver was Intel-only/Rosetta, so I wasn't sure where the issues were. I also couldn't figure out why it was working previously, WITHOUT having build 439 (before updating to build 4.8.2). Maybe I was on a previous beta build?

I've got Rosetta installed, and I just went ahead and installed the beta build, so I can indeed connect to Oracle servers again. TablePlus crashes about once an hour now, but it's a small price to pay for Oracle connectivity I suppose.

Thanks for the response!
